lib/graphql/argument.rb

```diff
@@ -21,6 +21,15 @@ class Argument
 
     ensure_defined(:name, :description, :default_value, :type=, :type)
 
+    def default_value?
+      !!@has_default_value
+    end
+
+    def default_value=(new_default_value)
+      @has_default_value = true
+      @default_value = new_default_value
+    end
```

Comment:
I guess there's no way to _unset_ a default value at this point, but I think that'll be fine, cross that bridge when we get to it